ui/grpc-web/net/grpc/gateway/backend/backend.cc

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Not sure how C files are used in our project. Can you please clarify?

Reply:
/grpc-web/ is our dependency. i checked in the entire project for the time being. i plan to remove this folder soon when build process is configured.